src/Css/Media.elm

```diff
@@ -0,0 +1,1168 @@
+module Css.Media
+    exposing
+        ( MediaQueryComponent
+        , MediaQuery
+        , media
+        , mediaQuery
+        , withMedia
+        , withMediaQuery
+        , only
+        , not
+        , or
+        , all
+        , screen
+        , print
+        , speech
+        , braille
+        , embossed
+        , handheld
+        , projection
+        , tty
```

Comment:
`tty`, `tv`, `projection`, `handheld`, `braille`, `embossed`, and `aural` [were all deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/@media#Media_types) and "shouldn't be used" apparently. 😄